### DISCUSSION TITLE
histoire.config.ts on root looks to not work

### DISCUSSION THREAD

**Discussion by snmatsui** [histoire-dev/histoire#431] [General]
I tried histoire on Docker container.
I placed package.json to container root and placed histoire.config.ts.
However, storyMatch config is ignored and scanned unintentionally
 files.

```
file:///node_modules/histoire/dist/node/stories.js:70
        throw new Error(`No support plugin found for file ${absoluteFilePath}`);
              ^

Error: No support plugin found for file /bin/dnsdomainname
    at addStory (file:///node_modules/histoire/dist/node/stories.js:70:15)
```

this issue loooks to be caused by this while condition.

https://github.com/histoire-dev/histoire/blob/main/packages/histoire/src/node/util/find-up.ts#L13

base image: node:16-slim

**Comment by josecl** [comment]
I had a similar problem with my `.env` files. Fixed with the `storyIgnored` parameter:

```js
storyIgnored: ['**/node_modules/**', '**/dist/**', '.env*'],
```
